pkg/manifests/manifests.go

```diff
@@ -726,7 +682,7 @@ func (f *Factory) PrometheusK8s(host string) (*monv1.Prometheus, error) {
 		}
 	}
 
-	if f.config.EtcdConfig == nil {
+	if f.config.EtcdConfig.Enabled != nil && *f.config.EtcdConfig.Enabled == false {
```

Comment:
that's a lot of duplicated `f.config.EtcdConfig.Enabled != nil && *f.config.EtcdConfig.Enabled == false` checks, can we implement this via a `nil` aware pointer receiver method on `EtcdConfig`?

Reply:
yeah, I was thinking that earlier. I just added it now :)